docs/internals/orientation-vector.md

```diff
@@ -1,43 +1,33 @@
 ---
-title: "Orientation Vector"
-linkTitle: "Orientation Vector"
+title: "Poses and Orientation Vectors"
+linkTitle: "Pose"
```

Comment:
Currently, this doc only covers orientation vector. If we want to cover all of pose (including the position part), more needs to be added. https://python.viam.dev/autoapi/viam/components/arm/index.html#viam.components.arm.Pose

Reply:
Gotcha I'll change it back.